docs/api/command-line-switches.md

```diff
@@ -272,6 +272,13 @@ Set the `host:port` to be used when the inspector is activated. Useful when acti
 
 Aliased to `--debug-port=[host:]port`.
 
+### `--disable-warning=code-or-type`
+
+Disable specific process warnings by `code` or `type`.
```

Comment:
I think it'd be useful to link to the node documentation on usage or provide our own examples (i.e. what are valid types and what's the format for the codes?)